### PR TITLE
Wake up

### PR DESCRIPTION
**Pull request by christianutama**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/1935)

### CODE REVIEW

**Comment by christianutama** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by christianutama** [comment]
Please review this, @casey-collab

File: english/marshmallow.md [hunk 1]
```diff
@@ -385,3 +385,7 @@ Do you:
 [Talk to Finn](talk-to-finn/finn.md)
 
 [Play Guitar](guitar/guitar.md)
+
+[Rinse the marshmallow under the tap then eat it](dysentry/dysentry.md)
```
Comment (casey-collab) [inline english/marshmallow.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by christianutama** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :ice_cream:

**Pull request merged**